app/views/address_info/new.html.erb

```diff
@@ -55,12 +55,12 @@
                     <div class="form-group row">
                         <div class="col-sm-6 mb-3 mb-sm-0">
                             <%= f.email_field :mail_id, autofocus: true, class: 'form-control form-control-user',
-                            placeholder: 'Enter your contact number' %>
+                            placeholder: 'Enter your mail id' %>
```

Comment:
@AkshayPashnathSuresh 
hard coding is not allowed. 
use locales (internationalization) .